Competition2018/src/main/java/competition/subsystems/elevator/commands/ElevatorMaintainerCommand.java

```diff
@@ -8,25 +9,29 @@
 import xbot.common.math.PIDManager;
 
 public class ElevatorMaintainerCommand extends BaseCommand {
-    
+
     ElevatorSubsystem elevator;
     PIDManager pid;
-    
+
     @Inject
     public ElevatorMaintainerCommand(ElevatorSubsystem elevator, PIDFactory pf) {
         this.elevator = elevator;
         pid = pf.createPIDManager("Elevator", 1, 0, 0);
         pid.setErrorThreshold(0.1);
     }
-    
+
     @Override
     public void initialize() {
-        log.info("Initializing");        
+        log.info("Initializing");
     }
 
     @Override
     public void execute() {
-        double power = pid.calculate(elevator.getTargetHeight(), elevator.currentHeight());
-        elevator.setPower(power);
+        if (elevator.isCalibrated()) {
+            double power = pid.calculate(elevator.getTargetHeight(), elevator.currentHeight());
+            elevator.setPower(power);
+        } else {
+            log.info("Elevator uncalibrated");
```

Comment:
If we get into this state, it seems that it will write messages into our log files 60+ times a second for the duration of the match... that's bad. Instead, how about we have it log once in "initialize" if this is the case and then no-op here?

Reply:
done